### PR TITLE
sort selector

### PR DESCRIPTION
**Pull request by z5160345**
## Context

https://www.pivotaltracker.com/story/show/175033816

## Description 💬
Added in a sort selector component that can be added anywhere. Implemented sort by title, artist and album.
Did some small code clean up:
Imports: react -> local files -> external packages
Code: imports -> function definition -> state -> constants -> functions -> return

## Checklist ✅:

- [X] I assigned the story to myself on Pivotal Tracker before I started working on this
- [X] Added a link to the user story context
- [X] Added a description of my changes
- [X] Assigned at least two reviewers

### CODE REVIEW

File: src/components/SortSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { Dispatch, SetStateAction, useState } from 'react'
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+
+export default function Subplaylist (props: {
+  setSort: Dispatch<SetStateAction<string>>
```
Comment (featherbear) [inline src/components/SortSelector.tsx]: Can we make this prop accept an unopinionated function callback; the current prop is very tied in to be implemented with a useState[1]-like value

**Review by featherbear — CHANGES_REQUESTED** [review]

File: src/components/SortSelector.tsx (hunk 1 above)
Comment (z5160345) [reply src/components/SortSelector.tsx]: Not sure if this is what you mean, but have a look at the new code

**Review by CarlinWilliamson — APPROVED** [review]
Pivotal says you should also have alphabetical and track number sorting.

The former makes zero sense. The later probably only makes sense within an album (i.e. after we've done grouping?)

File: src/components/SortSelector.tsx (hunk 1 above)
Comment (featherbear) [reply src/components/SortSelector.tsx]: Looks good now :)

**Review by featherbear — APPROVED** [review]

**Pull request merged**